config/sp_api.php

```diff
@@ -70,7 +70,7 @@
      * the configured URI as an extra layer of protection against making unintended calls to production
      * endpoints.
      */
-    'sandbox'                                   => env('SP_API_SANDBOX', true),
+    'sandbox'                                   => (bool) env('SP_API_SANDBOX', true),
```

Comment:
Yep basically! I was getting errors because in `phpunit.xml` I originally had this line which was being interpreted as a string:
```XML
<env name="SP_API_SANDBOX" value="1"/>
```
I thought about using `filter_var` like we did for the other issue with Amazon's `IsGift` etc, but decided instead to do a simple `(bool)` casting since this config file gets published to a developer's config file, and `filter_var` just looks too messy / seems non-Laravel-y.

Reply:
Nice good solution!